pkgs/tools/security/passphrase2pgp/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, pandoc, buildGoModule, fetchFromGitHub }:
```

Comment:
```suggestion
{ lib, buildGoModule, fetchFromGitHub }:
```

Reply:
Right. I thought about generating html, but after that decided against.